packages/nodejs/src/environment/NodeJsEnvironment.ts

```diff
@@ -144,9 +148,16 @@ function configureNetwork(env: Environment) {
 
     Boot.init(() => {
         if (config.installNetwork || (env.vars.boolean("nodejs.network") ?? true)) {
+            const basePathForUnixSockets = env.maybeGet(StorageService)?.location;
```

Comment:
Is that valid approach? If storage is "something else" (like Redis or such) then this might all be misleading. Also "location" there is right now mainly used for logging and not that much more, right?
Ok to have a TODO or we need to find a way to define if we have a real"FS storage" used or something else.

Reply:
I changed to use root.path environment variable which is what it should have been using originally.